src/test/java/com/google/cloud/spanner/pgadapter/python/django/DjangoBasicTest.java

```diff
@@ -230,4 +265,24 @@ public void testInsertAllTypesAllNull() throws IOException, InterruptedException
     assertEquals(1, mockSpanner.countRequestsOfType(ExecuteSqlRequest.class));
     assertEquals(sqlInsert, mockSpanner.getRequestsOfType(ExecuteSqlRequest.class).get(0).getSql());
   }
+
+  @Test
+  public void testSelectAllNull() throws IOException, InterruptedException {
```

Comment:
We should test that null values are correctly mapped for all possible data types, not only for strings. So we should create an ORM model / class that has all data types and then verify that we can select an instance of that model when:
1. All values are non-null
2. All values are null

See for example this test for `gorm`:
1. `gorm` struct for an entity with all data types: https://github.com/GoogleCloudPlatform/pgadapter/blob/8a19695373ed5371e9baea4a5980b10c50736403/src/test/golang/pgadapter_gorm_tests/gorm.go#L65
2. A test that verifies that selecting an instance of that entity with all null values works as expected: https://github.com/GoogleCloudPlatform/pgadapter/blob/8a19695373ed5371e9baea4a5980b10c50736403/src/test/java/com/google/cloud/spanner/pgadapter/golang/GormMockServerTest.java#L333

Reply:
done